pkg/sloop/webserver/webfiles.go

```diff
@@ -0,0 +1,54 @@
+package webserver
+
+import (
+	"fmt"
+	_ "github.com/jteeuwen/go-bindata"
+	"github.com/spf13/afero"
+	"html/template"
+	"path"
+	"sort"
+	"strings"
+)
+
+const (
+	prefix = "webfiles/"
+)
+
+// go-bindata -o bindata.go webfiles
+// ReadWebfile is a function which finds the webfiles that have been predefined and converted to binary format.
+// sample input : filepath= "webfiles/index.html"
+func readWebfile(filepath string, _, fs *afero.Afero) ([]byte, error) {
+	if !strings.HasPrefix(filepath, prefix) {
+		return []byte{}, fmt.Errorf("Webfile %v is invalid.  Must start with %v", filepath, prefix)
+	}
+	data, err := fs.ReadFile(filepath)
+	if err != nil {
```

Comment:
consider inverting this condition, returning success here; this allows the following code to become main line (not indented)